src/templates/sitemap.xml

```diff
@@ -116,7 +123,7 @@
         </loc>
         
         <lastmod>
-            2019-11-11
+            2019-11-12
```

Comment:
@bazzadp Today is the 10th. The md metadata has the 11th. How did the sitemap get the 12th?

Reply:
Well that's interesting!

However the md does not have the 11th. It is missing as it has two `published` dates and no `last_updated` (which is [my fault](https://github.com/HTTPArchive/almanac.httparchive.org/issues/149#issuecomment-552193999)). Can you correct that?

And will see if I can figure this out. But not able to repeat it now and it's only the JavaScript chapter that it's doing that for... which is weird. You defo don't have a weird edit in that locally do you?

Reply:
Aha! Fixed.

Reply:
I still don't understand how https://github.com/HTTPArchive/almanac.httparchive.org/pull/430/commits/2b68fc7f199298cd4843ec8ba069250a2b5ad60f caused it to be the 12? And still can't repeat it.